docs/docusaurus/docs/snippets/result_format.py

```diff
@@ -214,60 +214,3 @@
 context.add_or_update_expectation_suite(
     expectation_suite=test_suite,
 )
-
-
-# <snippet name="tests/integration/docusaurus/reference/core_concepts/result_format/result_format_checkpoint_example">
```

Comment:
this deletion was a manual change - snippet (and rest of file) not used anywhere

Reply:
@joshua-stauffer I found that the build failed without this. I think because of files that resulted from `invoke api-docs`. This part of the process and the gitignored directory are still pretty confusing to me

Reply:
wow, okay, great catch!